osbuild/util/parsing.py

```diff
@@ -31,3 +33,72 @@ def parse_size(s: str) -> Union[int, str]:
                 return "unlimited"
 
     raise TypeError(f"invalid size value: '{s}'")
+
+
+def parse_mount(url: ParseResult, args: Dict) -> os.PathLike:
+    """
+    Parses the mount URL to extract the root path.
+
+    Parameters:
+    - url (ParseResult): The ParseResult object obtained from urlparse.
+    - args (Dict): A dictionary containing arguments including mounts information.
```

Comment:
maybe add something like: ` A dictionary containing arguments including mounts and path information as passed by osbuild.api.arguments()` (here and in the other docstrings)